shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/BridgeExecutorServiceImpl.java

```diff
@@ -28,12 +29,15 @@
 import io.fabric8.kubernetes.api.model.Service;
 import io.fabric8.kubernetes.api.model.apps.Deployment;
 import io.fabric8.kubernetes.client.KubernetesClient;
+import io.quarkus.runtime.configuration.ProfileManager;
 
 @ApplicationScoped
 public class BridgeExecutorServiceImpl implements BridgeExecutorService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(BridgeExecutorServiceImpl.class);
 
+    private boolean isDev = ProfileManager.getActiveProfile().equals("minikube");
```

Comment:
It seems to me that this breaks when testing locally with `minikube` profile and an external managed kafka cluster. Wdyt?